spiel/st/m0t1fs_spiel_st.sh

```diff
@@ -212,17 +212,17 @@ m0d_with_rms_start() {
     local FI_OPTS="m0_ha_msg_accept:invalid_confc:always"
     local M0D_OPTS="$OPTS -o $FI_OPTS"
 
-    stub_confdb | $M0_SRC_DIR/utils/m0confgen >$CONF_FILE
+    stub_confdb | "$M0_SRC_DIR"/utils/m0confgen >"$CONF_FILE"
 
-    echo "--- `date` ---" >>$path/m0d.log
-    cd $path
+    echo "--- `date` ---" >>"$path"/m0d.log
+    cd "$path"
 
-    echo $M0_SRC_DIR/utils/mkfs/m0mkfs $OPTS
-    $M0_SRC_DIR/utils/mkfs/m0mkfs $OPTS >>$path/mkfs.log ||
+    echo "$M0_SRC_DIR"/utils/mkfs/m0mkfs "$OPTS"
+    "$M0_SRC_DIR"/utils/mkfs/m0mkfs "$OPTS" >>"$path"/mkfs.log ||
```

Comment:
The `$OPTS` should not be double quoted.

`$OPTS` is a list of options, separated by space/tab.
They should be passed to m0mkfs as it is.
If we use `"$OPTS"`, all arguments will be combined to a single one. This is wrong.

This will cause error.